src/app/core/core.state.ts

```diff
@@ -30,6 +39,33 @@ export const selectAuthState = createFeatureSelector<AppState, AuthState>(
   'auth'
 );
 
+export const selectRouterState = createFeatureSelector<
```

Comment:
Let's make this typed to stay consistent with the rest of the codebase

```
export const selectRouterState = createFeatureSelector<AppState, RouterReducerState>('router');
```

Reply:
Using this won't let provide the `CustomSerializer` in the selector.
So `state` in `(entities, params) => params && entities[params.state.params.id]` only contains `root` and `url`.

Reply:
Woops you're right, I meant: 

```
export const selectRouterState = createFeatureSelector<AppState, RouterReducerState<RouterStateUrl>>('router');
```

Reply:
Try changing the string parameter to `roter` or change the `router` property of the `Appstate` to `roter` and you'll notice it 😄 

It's makes sure the string you give with `createFeatureSelector` exists in the state tree.

Reply:
Okay, great! Thank you 😄